CMakeLists.txt

```
@@ -155,12 +155,12 @@ if(ANDROID)
   get_filename_component(QT_ANDROID_PATH_CMAKE_DIR_${ANDROID_ABI} "${Qt6_DIR}/.." ABSOLUTE)
   message(STATUS "QT_ANDROID_PATH_CMAKE_DIR_${ANDROID_ABI}: ${QT_ANDROID_PATH_CMAKE_DIR_${ANDROID_ABI}}")
 
-  set(QT_HOST_PATH "${VCPKG_INSTALLED_DIR}/x64-linux")
-  set(QT_HOST_PATH_CMAKE_DIR "${VCPKG_INSTALLED_DIR}/x64-linux/share")
+  set(QT_HOST_PATH "${VCPKG_INSTALLED_DIR}/x64-linux" CACHE PATH "Path to host qt prefix")
```

Hm this change will impact self built APKs through docker, and feels like a shame to let go of the vcpkg_installed_dir variable.

What about adding a vcpkg_installed_dir/vcpkg_host_triplet variable instead? Even if we have to define it, it's cleaner.

---

```suggestion
  set(QT_HOST_PATH "${VCPKG_INSTALLED_DIR}/${VCPKG_HOST_TRIPLET}")
```

@nirvn would that be better?

---

@m-kuhn , very much so